Source/Core/Core/IOS/Network/KD/NWC24DL.cpp

```diff
@@ -120,6 +120,11 @@ bool NWC24Dl::IsEncrypted(u16 entry_index) const
   return !!Common::ExtractBit(Common::swap32(m_data.entries[entry_index].flags), 3);
 }
 
+bool NWC24Dl::IsNotRSASigned(u16 entry_index) const
```

Comment:
I see that the bit being set does indeed mean 'not signed', but I think this function makes more sense if you invert it to 'is RSA signed'.